nixos/modules/virtualisation/lxd.nix

```diff
@@ -100,6 +100,10 @@ in
       packages = [ cfg.lxcPackage ];
     };
 
+    # TODO: remove once LXD gets proper support for cgroupsv2
+    # (currently most of the e.g. CPU accounting stuff doesn't work)
+    systemd.enableUnifiedCgroupHierarchy = false;
```

Comment:
Is there any upstream issue to mention here?

Reply:
Unfortunately, I haven't been able to find any tracking issue - though I've found e.g. https://github.com/lxc/lxd/pull/8146, which suggests that CPU accounting _should work_ - I guess it'll all be deployed with the next LXD's version.